### ISSUE TITLE
Support PDF and ePub 2/3 rendering as HTML

### ISSUE THREAD

**Issue by anjackson** [ukwa/ukwa-pywb#68]
The document access system must wrap the document in either [pdf.js](https://mozilla.github.io/pdf.js/getting_started/) or [epub.js](https://github.com/futurepress/epub.js) depending on format (is this possible in the theme? #67).  The result must meet or exceed the WCAG 2.1 AA accessibility standard. The pdf.js and ePub.js libraries are believed to meet this accessibility requirement, but any additional elements required in order to use those libraries within uwka-pywb must be engineered to the same standard.

The system should also block `Content-Dispostion: attachement` downloads, and should display a contact/help page if the document in question is not HTML or PDF or ePub.

It is envisaged that support for other formats may be added in the future, either via client-side or server-side format conversion to HTML.  This should be kept in mind when implementing this feature.

**Comment by ikreymer** [comment]
Initial draft of support for pdf.js and epub.js is part of #85. 

As mentioned in #74, content-disposition alone may not be enough to prevent downloads, if download prevention is needed for direct pywb access. If only loading from the viewer, there's a clear way to stop all downloads, which would be more fool-proof.

**Comment by anjackson** [comment]
This works pretty well.  Ideally, especially for printing, I'd like to experiment with some of ePub.js's other [rendering and flow options](https://github.com/futurepress/epub.js/#render-methods). e.g. [scrolled-doc](http://futurepress.github.io/epub.js/examples/scrolled.html) or possibly even [continuous-scrolled](http://futurepress.github.io/epub.js/examples/continuous-scrolled.html).

However, it looks to me like [ePub Reader doesn't make it easy to change this](https://github.com/futurepress/epubjs-reader/blob/master/reader/js/reader.js#L3285). So maybe we should push this off to a future iteration.

**Comment by anjackson** [comment]
As our original request has been implemented, I'm going to call this ticket done. We need some changes to the ePub view but we'll follow up those up under the follow-on support arrangement. See #100
